source/input.js

```diff
@@ -468,11 +472,15 @@ export default class Input extends Component
 	// `<input/>` `onKeyDown` handler
 	on_key_down(event)
 	{
+		const {onKeyDown} = this.props
+
 		// Expand country `<select/>`` on "Down arrow" key press
 		if (event.keyCode === 40)
 		{
 			this.select.toggle()
 		}
+
+		onKeyDown(event)
```

Comment:
If this property wasn't passed then your PR will break the component for everyone else